### PR TITLE
Replaces usage of GS components by ui-kit ones (apart of charts)

### PR DESCRIPTION
**Pull request by ivan-nejezchleb**
JIRA: ONE-4786

Except charts replaces imports from GS by imports from ui-kit where possible

---

Supported PR commands:

| Command         | Description            |
| --------------- | ---------------------- |
| `ok to test`    | Re-run standard checks |
| `extended test` | BackstopJS tests       |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] review was done by a Code owner [if necessary](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-tell-if-my-pull-request-needs-approval-by-a-code-owner) (if you think it is not necessary, explain the reasoning in the description or in a comment)
-   [x] `check` passes
-   [x] `check-extended` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

**Comment by ivan-nejezchleb** [comment]
extended test

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/1813/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/3420/ : SUCCESS in 14m 15s

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/1814/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/549/ : SUCCESS in 31m 21s

File: libs/sdk-ui-kit/src/Overlay/Overlay.tsx [hunk 1]
```diff
@@ -96,12 +93,9 @@ export class Overlay<T> extends React.Component<IOverlayProps<T>, IOverlayState>
             },
         };
 
-        this.isComponentMounted = false;
+        this.isComponentMounted = false; // TODO: WHY?
```
Comment (xMort) [inline libs/sdk-ui-kit/src/Overlay/Overlay.tsx:96]: This should not be merged imho.

**Comment by xMort** [comment]
Shouldn't you run "rush change" in this case?

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/1816/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/3424/ : SUCCESS in 11m 07s

**Comment by no23reason** [comment]
@xMort rush change is not necessarry here in my opinion, as all the significant changes are in components not marked as public and from the end user's POV this is more of an implementation detail.

**Review by no23reason — APPROVED** [review]
LGTM

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/582/ : SUCCESS in 31s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/843/ : SUCCESS in 26s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/3426/ : SUCCESS in 15m 02s

**Pull request merged**